l10n_fr_mis_reports/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+French MIS reports
+==================
+
+This modules provides MIS Builder Report templates for the French
+P&L and Balance Sheet taking the liasse fiscale as reference.
+
+Installation
+============
+
+This module depends on the *mis_builder* module, which is available in the OCA project `account-financial-reporting <https://github.com/OCA/account-financial-reporting>`.
+
+Configuration
+=============
+
+To configure this module, you need to go to
+*Accounting > Reporting > MIS Reports* and create a report instance
+according to the desired time periods and using one of the following
+templates provided by this module:
+
+* Compte de résultat français
+* Bilan comptable français
+
+To obtain correct results, the account codes prefixes must match the official
+French chart of accounts.
+
+Usage
+=====
+
+To use this module, you need to go to
+*Accounting > Reporting > MIS Reports* and use the buttons
+available on the previously configured reports such as Preview,
+Print, Export, Add to dashboard.
+
+Credits
```

Comment:
You should also add the "test me on runbot button".
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst